docs/guides/esp32/setup_idf_chip.md

```diff
@@ -16,7 +16,7 @@ step.
 -   [Linux](https://docs.espressif.com/projects/esp-idf/en/v5.3/esp32/get-started/linux-macos-setup.html#for-linux-users)
 -   [macOS](https://docs.espressif.com/projects/esp-idf/en/v5.3/esp32/get-started/linux-macos-setup.html#for-macos-users)
 
-### Get IDF v5.1.2
+### Get IDF v5.3
```

Comment:
Now that we are already here, can you please change it to `Get ESP-IDF v5.3`? Thanks!

Reply:
done ☺